jbmc/regression/jbmc/lazyloading_opaquereturn/Opaque.java

```diff
@@ -0,0 +1,6 @@
+public class Opaque
```

Comment:
Is this folder missing the script that deletes the opaque class?

Reply:
No - the `Opaque.class` is not checked in unlike the other `.class` files, do you think including a build script is worthwhile? I think it is in general not done but that's isn't an argument against starting.

Reply:
I missed that it was not included and was thinking of java-compile-tests.